apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -172,28 +147,28 @@ class SectionsPage extends Component {
             color={Button.ButtonColor.gray}
           />
         }
-        {sectionsLoaded && showGoogleClassroom &&
+        {asyncLoadComplete && showGoogleClassroom &&
```

Comment:
instead of a bunch of individual `asyncLoadComplete` should we have this all in a single div behind an `asyncLoadComplete`?